plugins/main/package.json

```diff
@@ -64,9 +67,7 @@
     "react-cookie": "^4.0.3",
     "read-last-lines": "^1.7.2",
     "timsort": "^0.3.0",
-    "typescript": "^5.0.4",
-    "dompurify": "^3.1.3",
-    "jsdom": "16.7.0"
+    "typescript": "^5.0.4"
```

Comment:
**nitpick**: this should be a development dependency.

Reply:
Done